README.md

```diff
@@ -1,6 +1,20 @@
-# idena-pocket
-A web wallet for Idena.
+# Idena-pocket
+```Idena-pocket wallet is available at https://pocket.idena.dev```
 
-## Links
-- [Documentation](https://www.idena.dev/idena-pocket)
-- [Telegram](https://t.me/idenadev)
+### Overview 🔎
+Idena-pocket is based on Idena-js library developed by [@idena-dev](https://github.com/idena-dev). 
+Idena-JS is built with typescript.
+
+This wallet does not need an active node needed to run.
```

Comment:
Can we avoid "need" repeat? Something like:"It doesn't need an active full node"